sdk/identity/identity/TROUBLESHOOTING.md

```diff
@@ -453,27 +457,39 @@ Once you've verified Azure PowerShell is using the correct account, validate tha
+
+## Troubleshoot `AzurePipelinesCredential` authentication issues
+
+| Error Message                                                                                                                                                                                                                                                                                                                                                            | Description                                                                                                                                                                       | Mitigation                                                                                                                                                                                                                                                                                                                                                                                                 |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| AADSTS900023: Specified tenant identifier '975f013f-7f24-47e8-a7d3-abc4752bf34' is neither a valid DNS name, nor a valid external domain.                                                                                                                                                                                                                                | The tenant id passed to the credential is invalid                                                                                                                                 | Verify the tenant id is valid. If the service connection was configured via a user-assigned managed identity, the tenant will be the one in which managed identity was registered. If the service connection is configured via a Service Principal, the tenant should be the one in which the Service Principal is registered.                                                                             |
```

Comment:
UAMIs each have a Service Principal. App Registrations have Enterprise Apps, each of which has a Service Principal. Do not use Service Principal as a name for App Registrations, it's not correct.

Reply:
done